src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,413 @@
+---
+title: Active areas of Ethereum research
+description: Explore different areas of open research and learn how to get involved.
+sidebar: true
+lang: en
+---
+
+# Open areas of Ethereum research
+
+One of the primary strengths of Ethereum is that it is constantly being improved by an active research and engineering community. There are many enthusiastic, skilled people worldwide that would like to apply themselves to outstanding issues in Ethereum, but it is not always easy to find out what those issues are. This page aims to outline some of the key areas of active research as a rough guide to Ethereum's cutting edge.
+
+## General research resources
```

Comment:
Barnabe & Francesco recommended adding an intro section on “how research works in ethereum” which I think is good idea. Notes here: https://notes.ethereum.org/wvg_V07BTPiSmg2AoUM2_Q?view#Open-research-content

Reply:
ok added a preliminary paragraph